src/script/mod.rs

```diff
@@ -118,10 +118,12 @@ impl Script {
     }
 
     // Used by PyScript
-    pub fn string_representation(&self) -> String {
+    // could I add an option flag here to include the length of a number (to be called from debugger to simpilfy the parsimg logic?)
+    pub fn string_representation(&self, include_byte_offsets: Option<bool>) -> String {
```

Comment:
`include_byte_offsets` should just be boolean, the Option<bool> just adds a layer of complexity we don't need